react.gradle

```diff
@@ -100,6 +100,18 @@ afterEvaluate {
         def nodeExecutableAndArgs = config.nodeExecutableAndArgs ?: ["node"]
         def extraPackagerArgs = config.extraPackagerArgs ?: []
 
+        def execCommand = []
+
+        if (config.cliPath || config.nodeExecutableAndArgs) {
+            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                execCommand.addAll(["cmd", "/c", *nodeExecutableAndArgs, cliPath])
+            } else {
+                execCommand.addAll([*nodeExecutableAndArgs, cliPath])
+            }
+        } else {
+            execCommand.addAll(["npx", "react-native"])
```

Comment:
Windows needs to run `npx.cmd` (see how it's done in CLI's native_modules.gradle)

Reply:
Fixed C: